src/mame/mr/dribling.cpp

```diff
@@ -544,14 +601,15 @@ ROM_END
 
 } // anonymous namespace
 
-
 /*************************************
```

Comment:
This blank line doesn’t need to be removed.

Reply:
Done